reflex/.templates/web/utils/state.js

```diff
@@ -101,7 +101,11 @@ export const getBackendURL = (url_str) => {
       } else if (endpoint.protocol === "http:") {
         endpoint.protocol = "https:";
       }
-      endpoint.port = ""; // Assume websocket is on https port via load balancer.
+      // If we are running on replit.dev, avoid deleting the port.
```

Comment:
nice. for context, in "development mode" repls are available at `<repl id>-<random string>.replit.dev`